app/views/downloads/_download_preface.html.erb

```diff
@@ -54,16 +54,22 @@ hide_freeform?(@work)) %>
     <% end %>
   <% end %>
 
-  <% unless @work.parents.blank? %>
-    <dl>
-      <dt><%= ts('This work was inspired by') %></dt>
-      <dd>
-        <% parent_work_list = @work.parent_work_relationships.reject{ |wr| wr.parent.nil? } %>
-        <%= parent_work_list.map{ |rw| "#{link_to(rw.parent.title,
-               url_for(action: :show, controller: rw.parent_type.underscore.pluralize, id: rw.parent_id,
-                       only_path: false)) } #{ts('by')} #{byline(rw.parent, only_path: false)}" }.join(", ").html_safe %>
-      </dd>
-    </dl>
+  <% related_works = @work.parent_work_relationships.reject{ |wr| !wr.parent } %>
+  <% if related_works.length > 0 %>
+    <ul>
+    <% related_works.each do |work| %>
+      <li>
+        <% relation_text = work.translation ?
+          "A translation of %{work_link} by %{author_link}" :
+          "Inspired by %{work_link} by %{author_link}"
+        %>
+        <% work_link = link_to work.parent.title.html_safe, work_url(work.parent) %>
```

Comment:
The related work can be an ExternalWork so we can't use `work_url` here, it should be a `url_for` like before.

Reply:
I like it a lot more than `url_for(action: :show, ...)` :+1: